File: php/URI/1118.php
<?php

$nota1;
$nota2;
$flag = 1;
$j = 0;

while ($flag == 1) {
    $nota1 = trim(fgets(STDIN));

    while ($nota1 > 10 || $nota1 < 0) {

        print("nota invalida\n");
        $nota1 = trim(fgets(STDIN));

    }

    $nota2 = trim(fgets(STDIN));
    while ($nota2 > 10 || $nota2 < 0) {

        print("nota invalida\n");
        $nota2 = trim(fgets(STDIN));

    }

    $media = ($nota1 + $nota2) / 2;
    echo "media = ". number_format($media,2,".", "", )."\n";

   echo "novo calculo (1-sim 2-nao)\n";
   $flag = trim(fgets(STDIN));

    while ($flag != 1 && $flag != 2) {
       echo"novo calculo (1-sim 2-nao)\n";
       $flag = trim(fgets(STDIN));

    }
}

?>

File: C/URI/1118.c
#include <stdio.h>

int main()
{

    double a, media;
    int cont, condicao;
    cont = 0;
    media = 0;
    condicao = 1;

    while (condicao != 2)
    {
        while (cont < 2)
        {
            scanf("%lf", &a);
            if (a >= 0 && a <= 10)
            {
                media = media + a;
                cont++;
            }
            else
                printf("nota invalida\n");
        }
        cont = 0;
        media = media / 2;
        printf("media = %.2lf\n", media);
        media = 0;
        while (condicao != 2 || condicao != 1)
        {
            printf("novo calculo (1-sim 2-nao)\n");
            scanf("%d", &condicao);
            if (condicao == 1 || condicao == 2)
            {
                break;
            }
        }
    }

    return 0;
}